### PR TITLE
RTL fixes for JSQ rewrite.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m [hunk 1]
```diff
@@ -55,7 +55,10 @@ - (void)updateMask
         return;
     }
     maskedSubview.frame = self.bounds;
-    [JSQMessagesMediaViewBubbleImageMasker applyBubbleImageMaskToMediaView:maskedSubview isOutgoing:self.isOutgoing];
+    // The JSQ masks are RTL-safe, so we need to invert the
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: typo? `s/are RTL-safe/are not RTL-safe/`

It is a good day when you get to xor.

File: Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m [hunk 2]
```diff
@@ -56,6 +56,7 @@ - (instancetype)init
         self.placeholderView.text = NSLocalizedString(@"new_message", @"");
         self.placeholderView.textColor = [UIColor lightGrayColor];
         self.placeholderView.userInteractionEnabled = NO;
+        self.placeholderView.textAlignment = NSTextAlignmentNatural;
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m]: Isn't this default? Why do we need to specify this?

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m (hunk 1 above)
Comment (charlesmchen) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: Done.

File: Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m (hunk 2 above)
Comment (charlesmchen) [reply Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m]: Removed.

**Pull request merged**